lib/libconfig.c

```diff
@@ -700,7 +702,12 @@ int config_write_file(config_t *config, const char *filename)
 
     if(fd >= 0)
     {
-      if(fsync(fd) != 0)
+#if defined(_WIN32)
+      int fsync_res = _commit(fd);
```

Comment:
if _commit() is identical in functionality to fsync() then why not just

#define fsync _commit

in the appropriate place in wincompat.h, and then delete the implementation of fsync() in wincompat.c.

Reply:
In fact remove wincompat.c altogether.